app/views/messages/ViewAllMessages.scala.html

```diff
@@ -48,45 +50,62 @@
     fullWidthContent = true
 ) {
 
-     @h1() {
-         @messages("viewAllMessages.heading", request.messageStatistics.countOfAllMessages)
-     }
+    @if(maybeDeletedMessageDescriptionKey.isDefined && maybeDeletedMessageDescriptionKey.get != ""){
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds govuk-!-padding-0">
```

Comment:
The notification banner isn't aligning to the left like all the other items on this page.
<img width="1281" alt="image" src="https://github.com/hmrc/emcs-tfe-frontend/assets/11269301/e3a864ec-30ee-4d2a-ae1c-f57c35b87c89">

I believe you could remove the first 2 div's which will then render this notification banner full width.

> The notification banner should be the same width as the page’s other content, such as components, headings and body text. For example, if the other content takes up two-thirds of the screen on desktop devices, then the notification banner should also take up two-thirds [https://design-system.service.gov.uk/components/notification-banner/]